frontend/pages/admin/IntegrationsPage/cards/MdmSettings/VppPage/components/AddVppModal/AddVppModal.tsx

```diff
@@ -70,14 +69,7 @@ const AddVppModal = ({ onCancel, onAdded }: IAddVppModalProps) => {
           graphicName="file-vpp"
           buttonType="link"
           buttonMessage={isUploading ? "Uploading..." : "Upload"}
-          filePreview={
-            tokenFile && (
-              <FileDetails
-                details={{ name: tokenFile.name }}
-                graphicName="file-vpp"
-              />
-            )
-          }
+          fileDetails={tokenFile ? { name: tokenFile.name } : undefined}
```

Comment:
Looks like the graphic name may have been accidentally removed here?

Reply:
It's passed in from line 69
<img width="855" alt="Screenshot 2024-09-16 at 8 01 43 AM" src="https://github.com/user-attachments/assets/3501550e-6394-4e52-8a25-92664d9edbfb">